src/viewer/Entity.tsx

```diff
@@ -85,6 +85,7 @@ class Entity extends React.Component<Props & CardsProps & PortalProps, State> {
 		}
 		const card = this.props.cards.getByDbfId(this.props.dbfId);
 
+		let gaEventTimeout: NodeJS.Timer;
```

Comment:
This is always a bit confusing, in Node.js `setTimeout` returns `NodeJS.Timer`, but as this is a browser, it will return `number`.